django_redis/cache.py

```diff
@@ -8,9 +8,11 @@
 from .util import load_class
 from .exceptions import ConnectionInterrupted
 
+
 DJANGO_REDIS_IGNORE_EXCEPTIONS = getattr(settings, "DJANGO_REDIS_IGNORE_EXCEPTIONS", False)
 DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = getattr(settings, "DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS", False)
 DJANGO_REDIS_LOGGER = getattr(settings, "DJANGO_REDIS_LOGGER", False)
+DJANGO_REDIS_DELETE_PATTERN_SCAN_COUNT = getattr(settings, "DJANGO_REDIS_DELETE_PATTERN_SCAN_COUNT", 10)
```

Comment:
I have actually found in some other function the library's code:
 
```
    def iter_keys(self, search, itersize=None, client=None, version=None):
        """
        Same as keys, but uses redis >= 2.8 cursors
        for make memory efficient keys iteration.
        """

        if client is None:
            client = self.get_client(write=False)

        pattern = self.make_key(search, version=version)
        for item in client.scan_iter(match=pattern, count=itersize):
            item = smart_text(item)
            yield self.reverse_key(item)
```
which makes me think that i will change this name to `DJANGO_REDIS_SCAN_ITERSIZE` to be more consistent with naming. What do you think?